versions.props

```diff
@@ -11,7 +11,8 @@ com.palantir.conjure.java:conjure-lib = 4.0.0
 com.palantir.conjure.java.api:* = 2.3.0
 com.palantir.conjure.verification:* = 0.16.7
 com.palantir.safe-logging:* = 1.10.1
-com.palantir.tracing:* = 3.1.0
+com.palantir.tracing:* = 3.2.0-rc4
```

Comment:
before merging, we need a real release of https://github.com/palantir/tracing-java/pull/124